### PR TITLE
feat: add young inspirers page

### PR DESCRIPTION
**Pull request by mattwang44**
## Types of changes

* **New feature**

## Description

Add a page for young inspirers event.

Design spec (Figma): [link](https://www.figma.com/file/1za4uYNpSonvgxdcLoHAzs/PyCon-Taiwan-2021-Web?node-id=166%3A0)
Web content: [link](https://hackmd.io/xFUBG77xT7-vTWk9mXBq7A?view)

### Preview
https://user-images.githubusercontent.com/24987826/128398787-c1b0aa3d-fa4b-47c2-8990-9c4a1bdde0d6.mp4

### CODE REVIEW

File: pages/conference/young-inspirers.vue [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+                >點我看直播</ext-link
+            ><br />
+            <span>線上共同筆記：</span
+            ><ext-link
+                href="https://hackmd.io/OXhLGazIRs6v4a_U_Zwq0g"
+                highlight
+                underline
+                >點我看共筆</ext-link
+            >
+        </ul>
```
Comment (flynnhou) [inline pages/conference/young-inspirers.vue:25]: Too messy and should not place `<p>` at the same level as `<li>` here.

It should be something like this:

```suggestion
        <ul class="list-disc">
            <li class="listTitle">活動時間：</li>
            <li class="list-none">2021/08/14 13:30-17:00</li>
            <li class="listTitle">活動形式：</li>
            <li class="list-none">
                Youtube 線上直播：
                <ext-link
                    href="https://youtu.be/SWnwRHgt-RA"
                    highlight
                    underline
                    >點我看直播</ext-link
                >
            </li>
            <li class="list-none">
                線上共同筆記：
                <ext-link
                    href="https://hackmd.io/OXhLGazIRs6v4a_U_Zwq0g"
                    highlight
                    underline
                    >點我看共筆</ext-link
                >
            </li>
        </ul>
```

File: pages/conference/young-inspirers.vue [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+                },
+            ],
+        }
+    },
+}
+</script>
+
+<style lang="postcss" scoped>
+.intro {
+    @apply text-xs md:text-sm font-serif mb-8;
```
Comment (flynnhou) [inline pages/conference/young-inspirers.vue:251]: `intro` should be `font-sans-serif`?

File: pages/conference/young-inspirers.vue [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+        </ul>
+
+        <div class="scheduleSection">
+            <h2>活動流程</h2>
+            <div class="table w-full mt-8">
+                <div class="table-row-group w-full">
+                    <div
+                        v-for="(item, index) in schedules"
+                        :key="`item_${index}`"
+                        class="table-row font-serif font-semibold"
```
Comment (flynnhou) [inline pages/conference/young-inspirers.vue:34]:
```suggestion
                        class="table-row font-sans-serif"
```

File: pages/conference/young-inspirers.vue (hunk 1 above)
Comment (flynnhou) [inline pages/conference/young-inspirers.vue:25]: Too messy and should not place `<p>` at the same level as `<li>` here.

It should be something like this:

```suggestion
        <ul class="list-disc">
            <li class="listTitle">活動時間：</li>
            <li class="list-none">2021/08/14 13:30-17:00</li>
            <li class="listTitle">活動形式：</li>
            <li class="list-none">
                Youtube 線上直播：
                <ext-link
                    href="https://youtu.be/SWnwRHgt-RA"
                    highlight
                    underline
                    >點我看直播</ext-link
                >
            </li>
            <li class="list-none">
                線上共同筆記：
                <ext-link
                    href="https://hackmd.io/OXhLGazIRs6v4a_U_Zwq0g"
                    highlight
                    underline
                    >點我看共筆</ext-link
                >
            </li>
        </ul>
```

**Review by flynnhou — CHANGES_REQUESTED** [review]

File: pages/conference/young-inspirers.vue (hunk 2 above)
Comment (mattwang44) [reply pages/conference/young-inspirers.vue]: I assume [`font-sans`](https://tailwindcss.com/docs/font-family#sans-serif) is for `sans-serif`?

Comment (flynnhou) [reply pages/conference/young-inspirers.vue]: Yeah. I forgot Tailwind's class name for sans-serif.

**Review by flynnhou — APPROVED** [review]
lgtm

**Pull request merged**